test/filter.spec.js

```diff
@@ -1144,5 +1144,32 @@ describe('resin-filter', () => {
 
       expect(sieve.filter(collection, input)).to.have.length(2)
     })
+
+    it('should behave correctly rule references a non-existent schema field', function () {
```

Comment:
This seems weird to me, would it make more sense like `...when a rule references...`?

Reply:
Yes you're right, this english is not good :(